catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/ActivityController.java

```diff
@@ -49,6 +49,12 @@
     // the OSGi Event Admin doesn't allow it.
     protected static final String ACTIVITY_TOPIC_COMETD = "/" + ActivityEvent.EVENT_TOPIC;
 
+    protected static final String ACTIVITY_TOPIC_COMETD_BROADCAST =
+            ACTIVITY_TOPIC_COMETD + "/broadcast";
+
+    protected static final String ACTIVITY_TOPIC_COMETD_DOWNLOADS =
+            ACTIVITY_TOPIC_COMETD + "/downloads";
```

Comment:
The activity status could be used as the channel segment since it's already required and it can only have one of six possible values.